common-npm-packages/docker-common-v2/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-tasks-docker-common-v2",
```

Comment:
I'd like to suggest to get rid of this package and publish it as a `azure-pipelines-tasks-docker-common` since there's one with this name published. What do you think of it?

cc: @anatolybolshakov

Reply:
Since `azure-pipelines-tasks-docker-common-v2` package is already published, I would prefer to update the same.

Reply:
`azure-pipelines-tasks-docker-common` is also published and I would suggest updating that one. Do you have any concerns?

Reply:
We don't plan any updates to `azure-pipelines-tasks-docker-common` as all our tasks are dependent on `azure-pipelines-tasks-docker-common-v2` now.

So the only package that is going to see updates is `azure-pipelines-tasks-docker-common-v2`. I don't see any value in updating 48 odd references, and I think it is okay for us to keep depending on `azure-pipelines-tasks-docker-common-v2` as this is inherited from the old versioning model for common packages.

cc: @ammohant @shigupt202